workflow/kfdrc_production_amplicon_architect.cwl

```diff
@@ -0,0 +1,286 @@
+   - `graph`: A text file for each amplicon listing the edges in the breakpoint graph, their categorization (sequence, discordant, concordant, source) and their copy counts
+   - `sv_pdf`: PDF image file displaying the SV view of AA
+   - `sv_png`: PNG image file displaying the SV view of AA
+
+  ### `tools/aa_classifier.cwl`
+  This tool classifies Amplicon Architect output to detect types of focal amplifications present
+  #### Inputs (critical):
+   - `data_repo`: Reference tar ball obtained from https://datasets.genepattern.org/?prefix=data/module_support_files/AmpliconArchitect/
+   - `data_ref_version`: enum, "GRCh38", "hg19", "GRCh37", "mm10", "GRCm38". Genome version in data repo to use. Should match `data_repo` downloaded.
+   - `cycles`: text file for each amplicon listing the edges in the breakpoint graph, their categorization (sequence, discordant, concordant, source) and their copy counts
+   - `graph`: A text file for each amplicon listing the edges in the breakpoint graph, their categorization (sequence, discordant, concordant, source) and their copy counts
+  #### Outputs:
+   - `amplicon_classification_profiles`: abstract classification of the amplicon
+   - `gene_list`: genes present on amplicons with each classification
+   - `ecDNA_cts`: not yet defined
+
+
+requirements:
+- class: MultipleInputFeatureRequirement
+- class: InlineJavascriptRequirement
+- class: ScatterFeatureRequirement
+
+inputs:
+  # Required
+  aa_data_repo: {type: File, doc: "Reference tar ball obtained from https://datasets.genepattern.org/?prefix=data/module_support_files/AmpliconArchitect/"}
+  aa_data_ref_version: {type: ['null', {type: enum, name: wgs_mode, symbols: ["GRCh38",
+          "hg19", "GRCh37", "mm10", "GRCm38"]}], doc: "Genome version in data repo\
+      \ to use", default: "GRCh38"}
+  tumor_align_file: {type: File, doc: "Tumor read alignment file. Can be cram or bam",
+    secondaryFiles: ['^.bai?', '.crai?']}
+  output_basename: {type: string, doc: "File name prefix for steps that require it"}
+  mosek_license_file: {type: File, doc: "This tool uses some software that requires\
+      \ a license file. You can get a personal or institutional one from https://www.mosek.com/license/request/."}
+  # If cram input and cnvkit not run
+  reference: {type: 'File?', doc: "fasta file, needed if cram input or cnv kit cnn\
+      \ not already built", secondaryFiles: [.fai]}
+  # If proper .cns file exists
+  cnvkit_cns: {type: 'File?', doc: ".cns file from previous CNVkit run, if available.\
+      \ DO NOT USE .call.cns"}
+  # If CNVkit needs running and .cnn file exists
+  cnvkit_cnn: {type: 'File?', doc: "If running using an existing .cnn, supply here"}
+  # If CNVkit needs running and .cnn NOT available
+  normal_align_file: {type: 'File?', doc: "Normal read alignment file. Can cram or\
+      \ bam", secondaryFiles: [{pattern: '^.bai', required: false}, {pattern: '.crai',
+        required: false}]}
```

Comment:
you're bouncing between notations here. i can tolerate one or the other but not both.

Reply:
Yes chef!